packages/relay/src/lib/precheck.ts

```diff
@@ -89,8 +89,14 @@ export class Precheck {
     const accountInfo = await this.mirrorNodeClient.getAccount(recoveredAddress, requestId);
 
     // @ts-ignore
-    if (accountInfo && accountInfo.ethereum_nonce > tx.nonce) {
-      throw predefined.NONCE_TOO_LOW;
+    if (accountInfo && accountInfo.ethereum_nonce) {
+      if (accountInfo.ethereum_nonce > tx.nonce) {
+        this.logger.trace(`${requestId} Failed nonce precheck for sendRawTransaction(tx.nonce=${tx.nonce}, accountInfo.ethereum_nonce=${accountInfo.ethereum_nonce})`);
+        throw predefined.NONCE_TOO_LOW;
+      } else if (accountInfo.ethereum_nonce + 1 < tx.nonce) {
```

Comment:
This solution seems worse. It will hard-wire a quarter-TPS submit rate.  Once you submit a TX you cannot submit another one until the first tx has not only been received by the consensus nodes, and processed by the consensus nodes, but the results have been processed by mirror node.

Reply:
Simplified

Reply:
Just logging for now so we get more info to decide if there's anything we can do